.eslintrc.js

```diff
@@ -19,31 +19,16 @@ module.exports = {
   },
 
   rules: {
-    // Require spacing around =>
-    "arrow-spacing": "error",
-
-    // Braces only needed for multi-line arrow function blocks
-    // "arrow-body-style": ["error", "as-needed"]
-
     // Always require spacing around a single line block
     "block-spacing": "error",
 
     // No newline before open brace for a block
     "brace-style": ["error", "1tbs", { allowSingleLine: true }],
```

Comment:
Should this be covered by prettier as well? I assume it would move the brace around?

Reply:
On further investigation, this should be dropped as well. This rule does match what Prettier does, but it's useless.